core/domain/feedback_services.py

```diff
@@ -172,3 +173,94 @@ def get_thread_analytics(exploration_id):
     """
     return feedback_jobs.FeedbackAnalyticsAggregator.get_thread_analytics(
         exploration_id)
+
+def create_suggestion(exploration_id, author_id, exploration_version,
+                      state_name, suggestion_content):
+    """Creates a new SuggestionModel object and the corresponding
+    FeedbackThreadModel object."""
+
+    thread_id = feedback_models.FeedbackThreadModel.generate_new_thread_id(
+        exploration_id)
+    (feedback_models.SuggestionModel
+        .create(exploration_id, thread_id, author_id, exploration_version,
+                state_name, suggestion_content))
+    thread = feedback_models.FeedbackThreadModel.create(
```

Comment:
The code looks similar to the existing `create_thread()` method in this file; it might be better to reuse it. Also, this doesn't create the initial message for the suggestion thread, which `create_thread()` does.

Maybe refactor so you have two public methods `create_thread()` and `create_suggestion()`, and these both make use of a private `_create_thread_model_and_first_message()`.